gitlab-base/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+# If in RHEL / UBI images, set ENV.
+# NOTICE: ubi-micro does not have grep
+if grep -q 'ID="rhel"' /etc/os-release ; then
```

Comment:
This will never be the case for our Wolfi-based images.

Reply:
True! I was copying their scripts literally. I wasn't sure if we are supposed to change them, but I know the answer now :D.